SingletonDesignPattern/Singleton.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace SingletonDesignPattern
+{
+    internal class Singleton
+    {
+        private string name;
+
+        //static instance creation
+        private static Singleton instance = new Singleton();
+
+        // private çonstructor
+        private Singleton()
+        {
+
+        }
+
+        // static method to get Instance
+        public static Singleton GetInstance()
+        {
+            return instance;
+        }
+
+        public string getValue()
+        {
+            return name;
+        }
+
+        public void setValue(string text)
```

Comment:
should be Pascal right?

Reply:
Yes, methods should be in PascalCase. Updated method names GetValue( ) and SetValue( ).